plugins/CoreHome/Controller.php

```diff
@@ -111,7 +111,7 @@ public function showInContext()
         $controllerName = Common::getRequestVar('moduleToLoad');
         $actionName     = Common::getRequestVar('actionToLoad', 'index');
 
-        if($controllerName == 'API') {
+        if($controllerName == 'API' || ($controllerName == 'Proxy' && $actionName == 'callApi')) {
             throw new Exception("Showing API requests in context is not supported for security reasons. Please change query parameter 'moduleToLoad'.");
```

Comment:
Yeah think too we wouldn't start it before 3.0. 2.X is in LTS mode and there shouldn't be too many changes. In 3.0 we can sometimes do this where it makes sense.